src/syntax_mapping.rs

```diff
@@ -1,6 +1,6 @@
 use std::path::Path;
 
-use crate::error::Result;
+use crate::{error::Result, ignored_suffixes::IgnoredSuffixes};
```

Comment:
Would you mind making `ignored_suffixes` a sub-module to `syntax_mapping` please? Similar to how it was a sub-module to `assets` before. Since it is not useful of its own, I think it makes sense to structure it as an implementation detail of syntax_mapping (which it is).

Reply:
I hope I got it right

Reply:
LGTM :+1: